### PR TITLE
Replaced functions to pass code reviews

### PR DESCRIPTION
**Pull request by boospot**
Hi, 
I have replaced the following functions with better functions to pass the code review by Codecanyon:

- replace function: `die()` with `wp_die()`
- replaced function: `json_encode()` with `wp_json_encode()`
- replaced function: `__()` with `esc_html__()`
- replaced function: `_x()` with `esc_html_x()`

### CODE REVIEW

File: inc/about/about.php [hunk 1]
```diff
@@ -150,7 +150,7 @@ public function enqueue() {
 	 */
 	public function change_footer_text() {
 		// Translators: %1$s - link to review form.
-		echo wp_kses_post( sprintf( __( 'Please rate <strong>Meta Box</strong> <a href="%1$s" target="_blank">&#9733;&#9733;&#9733;&#9733;&#9733;</a> on <a href="%1$s" target="_blank">WordPress.org</a> to help us spread the word. Thank you from the Meta Box team!', 'meta-box' ), 'https://wordpress.org/support/view/plugin-reviews/meta-box?filter=5#new-post' ) );
```
Comment (rilwis) [inline inc/about/about.php]: We don't need to escape here. `wp_kses_post` already escapes things.

File: inc/about/sections/support.php [hunk 2]
```diff
@@ -15,7 +15,7 @@
 			),
 		);
 		// Translators: %s - link to documentation.
-		echo wp_kses( sprintf( __( 'Still need help with Meta Box? We offer excellent support for you. But don\'t forget to check our <a href="%s">documentation</a> first.', 'meta-box' ), 'https://docs.metabox.io?utm_source=WordPress&utm_medium=link&utm_campaign=plugin' ), $allowed_html );
```
Comment (rilwis) [inline inc/about/sections/support.php]: `wp_kses` with `$allowed_html` does the escaping.

File: inc/about/sections/welcome.php [hunk 3]
```diff
@@ -11,7 +11,7 @@
 	$plugin = get_plugin_data( RWMB_DIR . 'meta-box.php', false, false );
 
 	// Translators: %s - Plugin name.
-	echo esc_html( sprintf( __( 'Welcome to %s', 'meta-box' ), $plugin['Name'] ) );
```
Comment (rilwis) [inline inc/about/sections/welcome.php]: This is wrong (twice escape).

File: inc/field.php [hunk 4]
```diff
@@ -351,7 +351,7 @@ public static function normalize( $field ) {
 				'clone'             => false,
 				'max_clone'         => 0,
 				'sort_clone'        => false,
-				'add_button'        => __( '+ Add more', 'meta-box' ),
```
Comment (rilwis) [inline inc/field.php]: Please add a space before `esc_html__`.

File: inc/fields/autocomplete.php [hunk 5]
```diff
@@ -20,7 +20,7 @@ public static function admin_enqueue_scripts() {
 			'rwmb-autocomplete',
 			'RWMB_Autocomplete',
 			array(
-				'delete' => __( 'Delete', 'meta-box' ),
```
Comment (rilwis) [inline inc/fields/autocomplete.php]: Please add a space before `esc_html__`

File: inc/fields/background.php [hunk 6]
```diff
@@ -61,7 +61,7 @@ public static function html( $meta, $field ) {
 				'type'        => 'file_input',
 				'id'          => "{$field['id']}_image",
 				'field_name'  => "{$field['field_name']}[image]",
-				'placeholder' => __( 'Background Image', 'meta-box' ),
```
Comment (rilwis) [inline inc/fields/background.php]: Please add a space before `esc_html__`

File: inc/fields/button.php [hunk 7]
```diff
@@ -31,7 +31,7 @@ public static function normalize( $field ) {
 		$field = wp_parse_args(
 			$field,
 			array(
-				'std' => __( 'Click me', 'meta-box' ),
```
Comment (rilwis) [inline inc/fields/button.php]: Please add a space before `esc_html__`

File: inc/fields/checkbox.php [hunk 8]
```diff
@@ -50,6 +50,6 @@ public static function input_description( $field ) {
 	 * @return string
 	 */
 	public static function format_single_value( $field, $value, $args, $post_id ) {
-		return $value ? __( 'Yes', 'meta-box' ) : __( 'No', 'meta-box' );
```
Comment (rilwis) [inline inc/fields/checkbox.php]: Please add a space before `esc_html__`

File: inc/fields/file.php [hunk 9]
```diff
@@ -21,9 +21,9 @@ public static function admin_enqueue_scripts() {
 			'rwmbFile',
 			array(
 				// Translators: %d is the number of files in singular form.
-				'maxFileUploadsSingle' => __( 'You may only upload maximum %d file', 'meta-box' ),
```
Comment (rilwis) [inline inc/fields/file.php]: Please add a space before `esc_html__`

File: inc/fields/file.php [hunk 10]
```diff
@@ -21,9 +21,9 @@ public static function admin_enqueue_scripts() {
...
+				'maxFileUploadsSingle' =>esc_html__( 'You may only upload maximum %d file', 'meta-box' ),
 				// Translators: %d is the number of files in plural form.
-				'maxFileUploadsPlural' => __( 'You may only upload maximum %d files', 'meta-box' ),
```
Comment (rilwis) [inline inc/fields/file.php]: Please add a space before `esc_html__`

File: inc/fields/file.php [hunk 11]
```diff
@@ -60,7 +60,7 @@ public static function ajax_delete_file() {
 		$field_value = self::raw_meta( $object_id, $field );
 		$field_value = $field['clone'] ? call_user_func_array( 'array_merge', $field_value ) : $field_value;
 		if ( ! in_array( $attachment, $field_value ) ) {
-			wp_send_json_error( __( 'Error: Invalid file', 'meta-box' ) );
```
Comment (rilwis) [inline inc/fields/file.php]: Please add a space before `esc_html__`

File: inc/fields/file.php [hunk 12]
```diff
@@ -74,7 +74,7 @@ public static function ajax_delete_file() {
 		if ( $result ) {
 			wp_send_json_success();
 		}
-		wp_send_json_error( __( 'Error: Cannot delete file', 'meta-box' ) );
```
Comment (rilwis) [inline inc/fields/file.php]: Please add a space before `esc_html__`

File: inc/fields/file.php [hunk 13]
```diff
@@ -87,7 +87,7 @@ public static function ajax_delete_file() {
 	 */
 	public static function html( $meta, $field ) {
 		$meta      = array_filter( (array) $meta );
-		$i18n_more = apply_filters( 'rwmb_file_add_string', _x( '+ Add new file', 'file upload', 'meta-box' ), $field );
```
Comment (rilwis) [inline inc/fields/file.php]: Please add a space before `esc_html_x`

File: inc/fields/file.php [hunk 14]
```diff
@@ -168,8 +168,8 @@ protected static function get_uploaded_files( $files, $field ) {
 	 * @return string
 	 */
 	protected static function file_html( $file, $index, $field ) {
-		$i18n_delete = apply_filters( 'rwmb_file_delete_string', _x( 'Delete', 'file upload', 'meta-box' ) );
```
Comment (rilwis) [inline inc/fields/file.php]: Please add a space before `esc_html_x`

File: inc/fields/file.php [hunk 15]
```diff
@@ -168,8 +168,8 @@ protected static function get_uploaded_files( $files, $field ) {
...
-		$i18n_edit   = apply_filters( 'rwmb_file_edit_string', _x( 'Edit', 'file upload', 'meta-box' ) );
```
Comment (rilwis) [inline inc/fields/file.php]: Please add a space before `esc_html_x`

File: inc/fields/input-list.php [hunk 16]
```diff
@@ -89,7 +89,7 @@ public static function get_attributes( $field, $value = null ) {
 	 */
 	public static function get_select_all_html( $field ) {
 		if ( $field['multiple'] && $field['select_all_none'] ) {
-			return sprintf( '<p class="rwmb-toggle-all-wrapper"><button class="rwmb-input-list-select-all-none button" data-name="%s">%s</button></p>', $field['id'], __( 'Toggle All', 'meta-box' ) );
```
Comment (rilwis) [inline inc/fields/input-list.php]: Please add a space before `esc_html__`

File: inc/fields/key-value.php [hunk 17]
```diff
@@ -132,8 +132,8 @@ public static function normalize( $field ) {
 		$field['placeholder']        = wp_parse_args(
 			(array) $field['placeholder'],
 			array(
-				'key'   => __( 'Key', 'meta-box' ),
```
Comment (rilwis) [inline inc/fields/key-value.php]: Please add a space before `esc_html__`

File: inc/update/settings.php [hunk 18]
```diff
@@ -93,7 +93,7 @@ public function render() {
 				<?php
 				printf(
 					// Translators: %1$s - URL to the My Account page, %2$s - URL to the pricing page.
-					wp_kses_post( __( 'To get the license key, visit the <a href="%1$s" target="_blank">My Account</a> page on metabox.io website. If you have not purchased any extension yet, please <a href="%2$s" target="_blank">get a new license here</a>.', 'meta-box' ) ),
```
Comment (rilwis) [inline inc/update/settings.php]: `wp_kses_post` already escapes string.

**Comment by rilwis** [comment]
I'll close this PR as it's old and not updated.

**Comment by boospot** [comment]
noted. thanks

**Pull request closed without merging**